lib/dpl/providers/pages/git.rb

```diff
@@ -46,10 +47,11 @@ class Git < Pages
              work_dir:            'Using temporary work directory %{work_dir}',
              committer_from_gh:   'The repo is configured to use committer user and email.',
              setup_dir:           'The source dir for deployment is %s',
+             target_dir:          'The target dir for deployment is %s',
              git_clone:           'Cloning the branch %{target_branch} from the remote repo',
              git_init:            'Initializing local git repo',
              git_checkout:        'Checking out orphan branch %{target_branch}',
-             copy_files:          'Copying %{src_dir} contents to %{work_dir}',
+             copy_files:          'Copying %{src_dir} contents to %{dst_dir}',
```

Comment:
i think this can just be `target_dir`, in which case we wouldn't need the extra method `dst_dir` below.